ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -499,8 +499,9 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
         std::vector<double>& /*local_b_data*/,
         LocalCoupledSolutions const& coupled_xs)
     {
+        auto const& particular_transport_process_id = coupled_xs.process_id;
```

Comment:
In my opinion, this is the only line (aside the vector and resize) relevant to the described changes.
```c++
auto const& transport_process_id = coupled_xs.process_id;
```
All of the renames can be dropped.